taskt/Core/Automation/Commands/DataTable Commands/RemoveDataRowCommand.cs

```diff
@@ -1,85 +1,85 @@
 ﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
+using System.Collections.Generic;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using System.Collections.Generic;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("DataTable Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you remove specified data rows.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to delete a specific row.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command attempts to delete a DataTable Row")]
+    [Group("DataTable Commands")]
+    [Description("This command removes specific DataRows.")]
```

Comment:
Nothing major but the descriptions of the others call out Datatables specifically.

Reply:
Fixed